src/helm/benchmark/static/schema_lite.yaml

```diff
@@ -106,33 +106,33 @@ models:
 
   # OpenAI
   - name: openai/text-davinci-003
-    display_name: text-davinci-003
+    display_name: TextDavinci003
```

Comment:
I don't see anyone calling it "TextDavinci003" on the internet (25 search results on Google). The official branding is just "text-davinci-003" - it's a GPT-3.5 so we can also call it "GPT-3.5 (text-davinci-003)"

see: https://platform.openai.com/docs/models/gpt-3-5